application/src/main/java/org/opentripplanner/service/realtimevehicles/RealtimeVehicleRepository.java

```diff
@@ -1,27 +1,20 @@
 package org.opentripplanner.service.realtimevehicles;
 
+import com.google.common.collect.Multimap;
 import java.util.List;
 import org.opentripplanner.service.realtimevehicles.model.RealtimeVehicle;
 import org.opentripplanner.transit.model.network.TripPattern;
 
 public interface RealtimeVehicleRepository {
   /**
-   * For the given pattern set all realtime vehicles.
+   * Stores the relationship between of realtime vehicles with a pattern for a given feed id.
```

Comment:
Isn't this used to store vehicles for multiples patterns at once?

Reply:
It is but each vehicle is only associated with a single pattern. Could you send a sentence that would make this absolutely clear to you?

Reply:
I have made another suggestion.